clearbit/examples_test.go

```diff
@@ -32,7 +32,7 @@ func ExampleNewClient_manuallyConfiguringEverything_output() {
 }
 
 func ExampleRevealService_Find_output() {
-	client := clearbit.NewClient()
+	client := clearbit.NewClient(clearbit.WithAPIKey(clearbitApiKey))
```

Comment:
you shouldn't need to do this, the client should configure itself:
https://github.com/clearbit/clearbit-go/blob/master/clearbit/clearbit.go#L64